src/AppBundle/Resources/config/content_view.yml

```diff
@@ -1,556 +1,242 @@
-                match: ~
-        embed:
-            ng_frontpage:
-                template: "@ezdesign/content/embed/ng_frontpage.html.twig"
-                match:
-                    Identifier\ContentType: ng_frontpage
-            ng_category:
-                template: "@ezdesign/content/embed/ng_category.html.twig"
-                match:
-                    Identifier\ContentType: ng_category
-            ng_landing_page:
-                template: "@ezdesign/content/embed/ng_landing_page.html.twig"
-                match:
-                    Identifier\ContentType: ng_landing_page
-            ng_article:
-                template: "@ezdesign/content/embed/ng_article.html.twig"
-                match:
-                    Identifier\ContentType: ng_article
-            ng_news:
-                template: "@ezdesign/content/embed/ng_news.html.twig"
-                match:
-                    Identifier\ContentType: ng_news
-            ng_blog_post:
-                template: "@ezdesign/content/embed/ng_blog_post.html.twig"
-                match:
-                    Identifier\ContentType: ng_blog_post
-            ng_audio:
-                template: "@ezdesign/content/embed/ng_audio.html.twig"
-                match:
-                    Identifier\ContentType: ng_audio
-            ng_gallery:
-                template: "@ezdesign/content/embed/ng_gallery.html.twig"
-                match:
-                    Identifier\ContentType: ng_gallery
-            ng_video:
-                template: "@ezdesign/content/embed/ng_video.html.twig"
-                match:
-                    Identifier\ContentType: ng_video
-            ng_htmlbox:
-                template: "@ezdesign/content/embed/ng_htmlbox.html.twig"
-                match:
-                    Identifier\ContentType: ng_htmlbox
-            ng_feedback_form:
-                template: "@ezdesign/content/embed/ng_feedback_form.html.twig"
-                match:
-                    Identifier\ContentType: ng_feedback_form
-            ng_shortcut:
-                template: "@ezdesign/content/embed/ng_shortcut.html.twig"
-                match:
-                    Identifier\ContentType: ng_shortcut
-            ng_banner:
-                template: "@ezdesign/content/embed/ng_banner.html.twig"
-                match:
-                    Identifier\ContentType: ng_banner
-            image:
-                template: "@ezdesign/content/embed/image.html.twig"
-                controller: "ngsite.controller.embed_view:embedImage"
-                match:
-                    Identifier\ContentType: image
-            file:
-                template: "@ezdesign/content/embed/file.html.twig"
-                match:
-                    Identifier\ContentType: file
-        search:
-            search:
-                template: "@ezdesign/content/search.html.twig"
-                match: ~
+ezpublish:
```

Comment:
I missed this, we reduced indenting https://github.com/netgen/media-site/pull/19, the file should now start with `frontend_group:`.